README.rst

```diff
@@ -44,6 +44,8 @@ Create a ``repos.yaml`` or ``repos.yml`` file:
         fetch_all:
             - oca
 
+    ./web: https://github.com/OCA/web.git 12.0
```

Comment:
By just reading the readme, I'm not sure what this means. Is it equivalent to?
```yaml
./web:
  remotes:
    origin: https://github.com/OCA/web.git
  merges:
    origin: 12.0
```

Reply:
Yeah, sorry about that. I know it's not clear but also didn't know where to describe it. Somehow it seemed not to be a feature that requires a section.

it's actually equivalent to:
```yaml
./web:
  remotes:
    origin: https://github.com/OCA/web.git
  merges:
    origin: 12.0
  target: origin 12.0
```